odoo/openupgrade/openupgrade_loading.py

```diff
@@ -157,6 +157,8 @@ def isrelated(model, k):
                     sorted([x[0] for x in v.selection]))
             else:
                 properties['selection_keys'] = 'function'
+        elif v.type == 'binary':
+            properties['attachment'] = str(getattr(v, "attachment", False))
```

Comment:
Please add the attribute here: https://github.com/OCA/OpenUpgrade/blob/10.0/odoo/addons/openupgrade_records/models/openupgrade_record.py#L39. It is safe to do so. The value of the field will not be compared and reported until we add the field to these lists here (which we should do in OpenUpgrade 11): https://github.com/OCA/OpenUpgrade/blob/10.0/odoo/addons/openupgrade_records/lib/compare.py#L190